src/framework/theme/components/select/select.component.html

```diff
@@ -1,4 +1,5 @@
 <button
+  [hidden]="isOptionSearchInputAllowed"
```

Comment:
why is it hidden? why not *ngIf?

Reply:
Just simpler. Nebular subscribes to some button events like focus. And with `*ngIf` we should implement unsubscription and resubscription mechanisms.  Looks too complicated for now